zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterOption.java

```diff
@@ -46,6 +49,17 @@ public void setHost(String host) {
     this.host = host;
   }
 
+  public boolean isSetPermission() {
```

Comment:
This is quite an ambiguous naming - usually in Java `set\get` used as a convention for getters and setters, so may be here we better call it like `permissionIsSet()` or `permissionWasSet()` - in this way we keep the almos-eglish readability in case of the code like

```java
if (permissionIsSet()) {
 //....
}
```